lighthouse-core/test/lib/sd-validation-test.js

```diff
@@ -17,7 +17,7 @@ describe('JSON validation', () => {
     `);
 
     assert.equal(errors.length, 1);
-    assert.equal(errors[0].path, 2);
```

Comment:
we should probably use strict equal here then, these would have been failing before I guess